docs/azure_arc_lighthouse/_index.md

```diff
@@ -0,0 +1,132 @@
+
+![Azure Arc extension management](./08.png)
+
+To work with Azure Arc extension management check out this Jumpstart scenarios:
+
+- [Deploy Monitoring Agent Extension on Azure Arc Linux and Windows servers using Extension Management](https://azurearcjumpstart.io/azure_arc_jumpstart/azure_arc_servers/day2/arc_vm_extension_mma_arm/#deploy-monitoring-agent-extension-on-azure-arc-linux-and-windows-servers-using-extension-management)
+- [Deploy Custom Script Extension on Azure Arc Linux and Windows servers using Extension Management](https://azurearcjumpstart.io/azure_arc_jumpstart/azure_arc_servers/day2/arc_vm_extension_customscript_arm/#deploy-custom-script-extension-on-azure-arc-linux-and-windows-servers-using-extension-management)
+- [Deploy Monitoring Agent Extension to Azure Arc Linux and Windows servers using Azure Policy](https://azurearcjumpstart.io/azure_arc_jumpstart/azure_arc_servers/day2/arc_policies_mma/)
+- [Integrate Azure Monitor for Containers with GKE as an Azure Arc Connected Cluster using Kubernetes extensions](https://azurearcjumpstart.io/azure_arc_jumpstart/azure_arc_k8s/day2/gke/gke_monitor_extension/)
+
+### Access Management
+
+Another important topic in unified operations is being able to keep track what the different teams are doing and provide insights into the operations that are perfomed on resources or subscriptions.
+
+With Azure Lighthouse the managed tenant can view Azure Activity log data to see all of the actions taken, this gives full visibility into operations that are performed through the Azure delegated resource management. Azure Lighthouse also uses supports Azure RBAC so you can give access to your resources following the principle of least priviledge so that users only have the permissions needed to complete their job.
+
+![Azure Lighthouse RBAC](./09.png)
+
+If you are centralizing operations in Azure and this includes on-premises and multicloud resources with Azure Arc, you will want to also control access to these services. Access to Azure Arc enabled resources is controlled by standard Azure RBAC, so from Access Control (IAM) in the Azure portal you can verify who has access to your resources and any actions taken from Azure will also be reflected on the Activity log.
+
+![Azure Arc RBAC](./10.png)
+
+The fact that Azure Arc enabled resources are part of ARM and access is controlled by Azure RBAC means that you can also have delegated management via Lighthouse with full visibility and insights on operations.
```

Comment:
```suggestion
Another important topic in unified operations is being able to keep track of what the different teams are doing and provide insights into the operations that are performed on resources or subscriptions.

With Azure Lighthouse the managed tenant can view Azure Activity log data to see all of the actions taken, this gives full visibility into operations that are performed through the Azure delegated resource management. Azure Lighthouse also uses supports Azure RBAC so you can give access to your resources following the principle of least privilege so that users only have the permissions needed to complete their job.

![Azure Lighthouse RBAC](./09.png)

If you are centralizing operations in Azure and this includes on-premises and multi-cloud resources with Azure Arc, you will want to also control access to these services. Access to Azure Arc enabled resources is controlled by standard Azure RBAC, so from Access Control (IAM) in the Azure portal you can verify who has access to your resources and any actions taken from Azure will also be reflected on the Activity log.

![Azure Arc RBAC](./10.png)

The fact that Azure Arc enabled resources are part of ARM and access is controlled by Azure RBAC means that you can also have delegated management via Lighthouse with full visibility and insights on operations.
```